doc/neps/nep-0026-missing-data-summary.rst

```diff
@@ -717,6 +717,16 @@ implementation strategy for NA dtypes.
 A further discussion overview page can be found at:
 https://github.com/njsmith/numpy/wiki/NA-discussion-status
 
+Additional References (2020):
+--------------------
+
+In 2018 Wes McKinney demonstrates, that sentinel values are actually less performant than bitmasks (contrary to popular belief also expressed here):
+https://wesmckinney.com/blog/bitmaps-vs-sentinel-values/
+
+In a follow-up Pasha Stetsenko optimizes the code for sentinel values by Wes McKinney, allowing them to perform similarly (but not better):
+https://github.com/st-pasha/microbench-nas/blob/master/README.md
+
+Note that the bitmasks used by Wes McKinnely are tightly packed and can not store additional information like "degree of missingness" mentioned as possible advantages of masks. For more details check out the Apache Arrow project, where this method is used.
```

Comment:
I'm not sure that this statement is appropriate in the "Additional References" section. Perhaps just a reference to Apache Arrow (preferably to the specific documentation related to bitmasks) would be more appropriate?